tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js

```diff
@@ -228,6 +241,96 @@ describe("Feature: Hub and Spoke", () => {
     });
   });
 
+  describe("Given a user opens a schema with hub required sections based on a repeating section", () => {
+    beforeEach("Load survey", async () => {
+      await browser.openQuestionnaire("test_hub_section_required_with_repeat.json");
+    });
+
+    it("When the repeating sections are complete. Then the hub should be displayed", async () => {
+      await $(PrimaryPersonListCollectorPage.yes()).click();
+      await $(PrimaryPersonListCollectorPage.submit()).click();
+      await $(PrimaryPersonListCollectorAddPage.firstName()).setValue("Marcus");
+      await $(PrimaryPersonListCollectorAddPage.lastName()).setValue("Twin");
+      await click(PrimaryPersonListCollectorAddPage.submit());
+      await $(ListCollectorPage.no()).click();
+      await $(ListCollectorPage.submit()).click();
+      await click(ListCollectorSectionSummaryPage.submit());
+
+      await $(ProxyRepeatPage.yes()).click();
+      await $(ProxyRepeatPage.submit()).click();
+      await $(DateOfBirthPage.day()).setValue(12);
+      await $(DateOfBirthPage.month()).setValue(4);
+      await $(DateOfBirthPage.year()).setValue(2021);
+      await click(DateOfBirthPage.submit());
+      await expect(browser).toHaveUrlContaining(HubPage.url());
```

Comment:
We should demo more than 1 repeating items. This could pass because it works when "at least" one repeat is complete. The test is "When all the repeating section are complete", emphasis on "all".

Reply:
Updated 👍